Makefile

```diff
@@ -1,3 +1,15 @@
+.PHONY: all
+local:
+	cargo build --release
+	strip target/release/miniserve
+	upx target/release/miniserve
```

Comment:
I suppose we shouldn't have that for local stuff and neither the strip.

Reply:
That's fair. Stripping would be a logical part of a `make install`, though. I don't want to get into setting up the stuff for `$PREFIX` to do that.